app/server/contactUsApi.ts

```diff
@@ -50,18 +51,39 @@ export const contactUsFormHandler = async (req: Request, res: Response) => {
     });
 };
 
-const parseAndValidate = (body: any): ContactUsReq | undefined => {
+const parseAndValidate = async (
+  body: any
+): Promise<ContactUsReq | undefined> => {
   try {
     const bodyAsJson = body ? JSON.parse(body) : "{}";
 
-    return validateContactUsFormBody(bodyAsJson)
+    const passedCaptchaValidation = await validateCaptchaToken(
```

Comment:
At this point in the execution it still hasn't been confirmed that `bodyAsJson.captchaToken` exists. `validateContactUsFormBody` is the validate step of this function, so it makes more sense to validate the captcha inside that function.